edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -78,6 +81,20 @@ def configured_jwt_decode_handler(token):
     return api_setting_jwt_decode_handler(token)
 
 
+def get_asymmetric_only_jwt_decode_handler(token):
+    """
+    Returns a jwt_decode_handler that will only validate asymmetrically signed JWTs.
+
+    WARNING: This will only work with a service that is configured to use the
+       jwt_decode_handler from this library. This can be used to decode an
+       already decoded JWT, to ensure it is asymmetrically signed. This check
+       can go away once the DEPR for symmetrically signed JWTs is complete:
+       https://github.com/openedx/public-engineering/issues/83
+    """
+    api_setting_jwt_decode_handler = api_settings.JWT_DECODE_HANDLER
+    return api_setting_jwt_decode_handler(token, decode_symmetric_token=False)
```

Comment:
Here we won't use the decoder from the setting.
```suggestion
    return jwt_decode_handler(token, decode_symmetric_token=False)
```